include/boost/beast/http/serializer.hpp

```diff
@@ -62,7 +62,7 @@ class serializer
 
     static_assert(is_body_writer<Body>::value,
         "BodyWriter requirements not met");
-
+    using body_value_type = typename Body::value_type;
```

Comment:
tsk tsk... you can't just casually add a public nested type! This affects the public API. As library developers we have to prefer to be inconvenienced over adding unnecessary interfaces.